src/runtime.js

```diff
@@ -87,6 +88,7 @@ export function installRuntimeServices(global) {
   installViewerService(global);
   installViewportService(global);
   installHistoryService(global);
+  installStorageService(global);
```

Comment:
I'm not sure that works with the current promise semantics. They must be
registered by the time a potential user calls.

On Wed, Sep 14, 2016 at 9:26 AM, Dima Voytenko <notifications@github.com>
wrote:

> In src/runtime.js
> <https://github.com/ampproject/amphtml/pull/4944#discussion_r78783975>:
>
> > @@ -87,6 +88,7 @@ export function installRuntimeServices(global) {
> >    installViewerService(global);
> >    installViewportService(global);
> >    installHistoryService(global);
> > +  installStorageService(global);
>
> Definitely. Would you be in favor of putting it into a micro-task for now
> and moving back once we register factories instead of instances?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ampproject/amphtml/pull/4944/files/38ae18a72ba2f5a6a8e5dbfb212ee19888c057fa#r78783975>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFeT1cKMSv5S6vO8-YBbcptFs2TTpI9ks5qqCAsgaJpZM4J7GGg>
> .
>